### PR TITLE
equinixmetal: use metro instead of facility

### PR DESCRIPTION
**Pull request by tormath1**
Follow up of: https://github.com/flatcar-linux/mantle/pull/281

Signed-off-by: Mathieu Tortuyaux <mtortuyaux@microsoft.com>

- [ ] Not sure if it's require a changelog entry ? Changelog entries added in the respective `changelog/` directory (user-facing change, bug fix, security fix, update)

### CODE REVIEW

File: jenkins/kola/packet.sh [hunk 1]
```diff
@@ -58,13 +58,13 @@ if [[ "${BOARD}" == "amd64-usr" ]] && [[ "${cl_internet_included}" != ""  ]]; th
...
     --gce-json-key="${UPLOAD_CREDS}" \
     --packet-api-key="${PACKET_API_KEY}" \
-    --packet-facility="${PACKET_REGION}" \
     --packet-image-url="${IMAGE_URL}" \
     --packet-installer-image-kernel-url="${KERNEL_URL}" \
     --packet-installer-image-cpio-url="${CPIO_URL}" \
     --packet-project="${PACKET_PROJECT}" \
     --packet-storage-url="${UPLOAD_ROOT}/mantle/packet" \
     --packet-plan="${INSTANCE}" \
+    --equinixmetal-metro="${PACKET_REGION}" \
```
Comment (pothos) [inline jenkins/kola/packet.sh:67]: We also need to change sv15 to the metro value in the Jenkins groovy file

Comment (tormath1) [reply jenkins/kola/packet.sh:67]: yep, just opened the PR

**Review by pothos — APPROVED** [review]

**Comment by pothos** [comment]
Please cherry pick for all active release maintenance branches

**Comment by pothos** [comment]
(No changelog for our release notes needed, it's an internal detail where the test ran)

**Comment by tormath1** [comment]
cherry-picked to:
* flatcar-3033
* flatcar-3066
* flatcar-3127
* flatcar-lts-2605

**Pull request merged**